default.conf

```diff
@@ -11,6 +11,14 @@ server {
     # This will enforce HTTP browsing into HTTPS and avoid ssl stripping attack
     add_header Strict-Transport-Security "max-age=31536000; includeSubdomains;";
 
+    # Add GZIP compression for speed optimization
+    gzip on;
```

Comment:
what about brotli?

Reply:
Could work, but this requires custom compilation of NGINX. Think this is a good first step.